pkgs/by-name/tt/tt/package.nix

```
@@ -0,0 +1,30 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
```

I ran this through `nix run github:NixOS/nixfmt` and the following changes were suggested. Apologies if I advised a leading comma before, I wasn't aware of a change in preference. 
```suggestion
{
  lib,
  fetchFromGitHub,
  buildGoModule,
```